app/models/host_status/configuration_status.rb

```diff
@@ -78,7 +78,7 @@ def to_status(options = {})
     end
 
     def relevant?
-      host.configuration?
+      host.configuration? || last_report.present?
```

Comment:
Can do, I've pushed another commit to add a setting.  It works slightly differently to 1.9 still, as the no reports status means this shows as a warning state for the host, rather than the greyed out state previously.